install/configure_administrative_access.md

```diff
@@ -159,7 +160,23 @@ Password read from Vault matches what was written
 SUCCESS
 ```
 
-## 8. Configure management nodes with CFS
+## 8. iSCSI SBPS configuration
+
+In CSM 1.6, all the worker nodes were configured and enabled as iSCSI SBPS targets. In CSM 1.7.0, selective
+node personalization is supported. All worker nodes are still **configured** for iSCSI, but selective node
+personalization gives administrators control over which worker nodes are actually enabled as iSCSI SBPS targets.
+The default behavior is still the same as in CSM 1.6, so if no action is taken to use this feature, then all
+worker nodes will be enabled as iSCSI targets.
+
+For administrators who do not wish to use this feature, no action is required, and this step can be skipped.
+Otherwise, before proceeding with the install, follow the procedure in the [CSM install](../operations/iscsi_sbps/Managing_selective_node_personalization.md#csm-install) section of [Managing Selective Node Personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md).
```

Comment:
Break up long line.
```suggestion
Otherwise, before proceeding with the install, follow the procedure in the
[CSM install](../operations/iscsi_sbps/Managing_selective_node_personalization.md#csm-install) section of
[Managing Selective Node Personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md).
```